### PR TITLE
Trial Tribute page

### PR DESCRIPTION
**Pull request by rotimiazeez**
# Tribute Page 🎉 

> This project is an upgrade to the hello microverse project, for upgrading HTML and CSS skills.

Additional description of the project and its features.
- This is part of the admission trial project but has been made more semantic.
- This project is reformed and documented more professionally.

## Built With ❤️ 

- HTML (with semantic tags)
- CSS (box-model)
- Stylelint, Github Actions

### CODE REVIEW

**Comment by Davidosky007** [comment]
Hi @rotimiazeez, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code. :pray: 

 ## Invalid CRR
_Check the comments under the review._

- [x] You have submitted a CRR without making a review at all [Please check this link on how to give a code review](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md)

You should make sure you perform a review on your code using Github features [Check project requirements on how](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/review_your_project.md)
 This Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rotimiazeez** [comment]
@Davidosky007 
Thanks for pointing this out, I really appreciate it and will work on it now. 👍🏽

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Hello World
+# Tribute Project
```
Comment (rotimiazeez) [inline README.md:3]: Good job 👍🏽

File: index.html [hunk 2]
```diff
@@ -9,9 +9,34 @@
...
+    <div class="main">
+        <div class="title text-align-center">
+          <h1>Dr. Norman Borlaug</h1>
+        </div>
+        <section>
+        <div class="img-div" >
+          
+          <img
+            src="https://assets-global.website-files.com/5dbb30f00775d4c32191a4df/5e31d5dc2639f85bc70363bd_Microverse-homepge-500w-tab-tiny-bq.png"
+            alt=""
```
Comment (rotimiazeez) [inline index.html:21]: I think the image should have an alternative text here 😄 .

**Review by rotimiazeez — COMMENTED** [review]
### Great job here! 👏🏽 
Here are some things you did right. 👇🏽 
- Use of semantic HTML tags 👍🏽,
- There are no linter errors 💪🏽 
- The project is well documented 🚀 

Keep up the good work. ❤️

File: README.md (hunk 1 above)
Comment (vichuge) [reply README.md:3]: - [x] Could you explain more about why is this a good job? Remember that a good point needs a clear explanation 😄

File: index.html (hunk 2 above)
Comment (vichuge) [reply index.html:21]: - [x] A way to improve this is to explain why this text is important. I kindly invite you to add it. 🐶

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

# Optional improvements
- In your comments, I add another comment to make better your code review.
- Why not use a template as we use it? This will improve your CR presentation a lot.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rotimiazeez** [comment]
@vichuge 👋🏽 
Thanks for approving my project 💪🏽 . Thanks for the comment as well, I'll work on it 👍🏽.

**Pull request merged**